scripts/apps/search/directives/ItemList.js

```diff
@@ -808,9 +811,13 @@ export function ItemList(
             });
 
             var ListItemInfo = function(props) {
+                var itemInfoClassName = 'item-info';
+                if (config.reducedItemListRowHeight) {
+                    itemInfoClassName = 'item-info item-info-reduced-rowheight';
+                }
                 return React.createElement(
                     'div',
-                    {className: 'item-info'},
+                    {className: itemInfoClassName},
```

Comment:
Sorry, ignore my above comment, you'll have to use `classNames` library here, which is already included in the project and is a common React pattern.

At the top of the file `import cx from 'classnames';`

```js
{className: cx('item-info', {'item-info-reduced-rowheight': config.list.narrow}),
```